include/fe/fe_base.h

```diff
@@ -250,6 +250,10 @@ class FEGenericBase : public FEAbstract
    * \returns The curl of the shape function at the quadrature
    * points.
    */
+
+#ifdef LIBMESH_ENABLE_INFINITE_ELEMENTS
```

Comment:
Should we possibly move the definition of the `virtual_for_inffe` macro to fe_base.h (or some other header) so that it can be used here as well?

Reply:
I forgot this part; but of course.
In this case, if I want to add changes, I use `jwpeterson:pr_2506` as reference for ``pull``/``push``

Reply:
Yeah, you should be able to fetch my branch, make changes, and push... let me know if you run into any issues with that.